src/Components/Blazor/Build/testassets/razorclasslibrary/wwwroot/exampleJsInterop.js

```diff
@@ -0,0 +1,8 @@
+// This file is to show how a library package may provide JavaScript interop features
```

Comment:
Could we avoid checking in the png file ^? Would it be sufficient to verify css \ js content files are copied around?

Reply:
Done